_posts/2019-11-26-october-2019.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: October 2019 Release Notes
+category: releases
+tags: release-notes
+---
+
+## Improvements
+
+* **Login.gov:** We continued to work on our integration with Login.gov.
+* **Elasticsearch Indexing:** We upgraded the Elasticsearch stack for custom indexing.
```

Comment:
It just occurred to me that "custom indexing" won't be meaningful for our customers. Would you please change this to something like, the following, including perhaps links to the help pages for each of these features? 
"We upgraded the underlying Elasticsearch search engine for our optional search features, such as Best Bets, RSS Feeds, Twitter, Federal Register, Health Topics, and Related Searches."